framework/field_functions/field_function_grid_based.cc

```diff
@@ -155,8 +154,7 @@ FieldFunctionGridBased::MakeSpatialDiscretization(const InputParameters& params)
     if (cs == "spherical") q_order = QuadratureOrder::FOURTH;
   }
 
-  if (sdm_type == "PWLC")
-    return PWLC::New(*grid_ptr, q_order, cs_type);
+  if (sdm_type == "PWLC") return PWLC::New(*grid_ptr, q_order, cs_type);
   else if (sdm_type == "PWLD")
     return PWLD::New(*grid_ptr, q_order, cs_type);
```

Comment:
this clang format is ugly. it was better before! I think you can halt clang-format. You can just add:
// clang-format off
    unformatted_code 
// clang-format on
to keep as it was before. We have this in several places

Reply:
I'm not totally sure of the clang-format option, but I've felt this way a few times before on this. I wouldn't be opposed to making all if statements go to the next line.

Reply:
There is one for that. I can send a PR with the change.